Examples/Framework/src/EventData/MeasurementCalibration.cpp

```diff
@@ -30,19 +31,12 @@ void ActsExamples::PassThroughCalibrator::calibrate(
   assert((idxSourceLink.index() < measurements.size()) &&
          "Source link index is outside the container bounds");
 
-  std::visit(
-      [&trackState](const auto& meas) {
-        using MeasurementType = std::decay_t<decltype(meas)>;
-        constexpr std::size_t size = MeasurementType::size();
+  const auto& meas = measurements[idxSourceLink.index()];
 
-        trackState.allocateCalibrated(meas.size());
-        assert(trackState.hasCalibrated());
-
-        trackState.calibrated<size>() = meas.parameters();
-        trackState.calibratedCovariance<size>() = meas.covariance();
-        trackState.setProjector(meas.projector());
-      },
-      measurements[idxSourceLink.index()]);
+  trackState.allocateCalibrated(meas.size());
+  trackState.effectiveCalibrated() = meas.parameters();
+  trackState.effectiveCalibratedCovariance() = meas.covariance();
```

Comment:
This could still be done on compile-time dimension numbers.

Reply:
Switched to assignment with template unrolling.